src/utils/api.js

```diff
@@ -274,18 +273,8 @@ const upload = (zipPath, appDir) => {
         }
       });
     })
-    .then(appVersion => {
+    .then(() => {
       printDone();
```

Comment:
If I'm reading this correctly, we're no longer running this version check on the `push`/`upload`, only on `version`, and we also no longer provide the URL to help update the dependencies, or understand what they are and why they're different.

Adding the check and URL on these commands was made via UX research/review so unless we found out something more, I don't feel very comfortable removing that as it might re-create confusion. #recommendation

Reply:
Sure, so there's two different (similar) pieces of code being run. in `zapier -v`, it previously spit out the node version and CLI version. I expanded that to include the depended-on core version (convenient) and figured I'd expand it to do a check for whether the installed version matches the required version. This is simply informational and only run on demand.

Separately, `test` (but probably soon `build` per discussion here) checks to see if the required version is behind the most recently published version and once a week (when `build` is run) it'll notify them that an update is available. The main goal of this PR was to reduce the amount of times we told them they were out of date (per feedback from [this typeform](https://admin.typeform.com/form/yPC7uY/results#responses)). Especially when the code is running fine, there's no particular need to upgrade and us pushing that aggressively devs (`core` on every upload, daily for `CLI`) was causing friction.

Reply:
Ok, that makes sense. Thank you for the explanation. 🙇 

I think we'd be more inline with those objectives if we are more explicit on showing them how to update what (like a link to a doc, similar to what we had) on `build`.

I'm not entirely sold on the benefits of splitting the `core` vs `cli` check, but we don't really have a lot of data on it, so I'm fine rolling with it, as you've been more involved in the recent research than I have.

Reply:
Ah, the reason they need to be in separate spots is because CLI can be run independently of an app. the CLI check can conveniently be put in `entry` which is run every time the CLI is invoked. We have to attach the check for `core` somewhere where we know we're working with an app. Your suggestion of `build` is a good one, since that's probably the most commonly run command.

Reply:
I see, that sounds like a great plan, then!